docs/content/latest/api/ycql/expr_fcall.md

```diff
@@ -60,6 +61,28 @@ function_call ::= function_name '(' [ arguments ... ] ')'
 | MIN | Returns the minimum value of column values |
 | MAX | Returns the maximum value of column values |
 
+
+## Partition_hash function
+`partition_hash` is a function that takes as arguments the values of partition keys of a row and returns an `uint16` hash value.
```

Comment:
that takes as arguments the values of the partition keys of a row
-->
that takes as arguments the partition key columns of the primary key of a row and returns a 'uint16' hash value representing the hash value for the row used for partitioning the table.